build.sbt

```diff
@@ -3441,6 +3448,7 @@ lazy val `runtime-compiler-dump-igv` =
   (project in file("engine/runtime-compiler-dump-igv"))
     .enablePlugins(JPMSPlugin)
     .settings(
+      customFrgaalJavaCompilerSettings("21", true),
```

Comment:
- adjusting #12061 
- Making sure sources of `runtime-compiler-dump-igv` are visible to `enso4igv` tooling.